README.md

```diff
@@ -1,13 +1,123 @@
 
 It uses plugins (drivers) to connect to different services.
 
-![cf-usb](https://region-b.geo-1.objects.hpcloudsvc.com/v1/11899734124432/imgs/usb.png)
+### Related Projects
+
+* [cf-usb-plugin](https://github.com/SUSE/cf-usb-plugin)
+* [cf-usb-sidecar](https://github.com/SUSE/cf-usb-sidecar)
+
+## Configuration and Usage
+
+The Universal Service Broker has three main components:
+
+* Universal Service Broker
+* Sidecars
+* cf CLI plugin
+
+The USB runs as a component within Cloud Foundry. Its [BOSH release](https://github.com/SUSE/cf-usb/tree/develop/cf-usb-release)
+is inside the USB main repo. This is included as a submodule in the SUSE CF project, and runs by default.
+
+These usage instructions assume:
+* A working SUSE CAP instance is available
+* The SUSE CAP instance is able to connect to a Docker registry
+* The user is able to access the Kubernetes cluster with the `kubectl` tool
+* The user has admin access to the SUSE CAP cluster
+
+### Sidecar Setup
+
+The USB itself is just a broker, and doesn't run any actual services. These are
+provided by the [sidecars](https://github.com/SUSE/cf-usb-sidecar/tree/develop/csm-extensions/services), and run outside of the CF cluster.
+
+To build the sidecar, check out the sidecar project:
+
+```
+mkdir -p $GOPATH/src/github.com/SUSE
+git clone https://github.com/SUSE/cf-usb-sidecar $GOPATH/src/github.com/SUSE/cf-usb-sidecar
+cd $GOPATH/src/github.com/SUSE/cf-usb-sidecar
+```
+
+Configure your Docker repository information:
+
+```
+export DOCKER_REPOSITORY=docker.io
+export DOCKER_ORGANIZATION=splatform
+
+docker login # if authorization is required
+```
+
+Then build the top level dependencies and the sidecar:
+
+```
+make tools
+make build-image
+cd csm-extensions/services/dev-mysql
+make build-image public-image helm
+```
+
+The generated Helm chart will be available in the `chart/` directory.
+
+Install the Helm chart:
+
+```
+# You will need to know the namespaces and domain for your cluster:
+UAA_NAMESPACE=uaa
+CF_NAMESPACE=cf
+CF_DOMAIN=cf-dev.io
+
+SIDECAR_NAMESPACE=mysql
+
+UAA_CA_CERT="$(kubectl get secret secret --namespace ${UAA_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_CA_CERT="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_PASSWORD="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['cluster-admin-password']}" | base64 --decode -)"
+
+helm install ./chart --name mysql-instance --namespace ${SIDECAR_NAMESPACE} \
+	--set "env.UAA_CA_CERT=${UAA_CA_CERT}" \
+	--set "env.CF_CA_CERT=${CF_CA_CERT}" \
+	--set "env.SERVICE_LOCATION=http://cf-usb-sidecar-mysql.${SIDECAR_NAMESPACE}.svc.cluster.local:8081" \
+	--set "env.SERVICE_MYSQL_HOST=AUTO" \
+	--set "env.CF_ADMIN_USER=admin" \
+	--set "env.CF_ADMIN_PASSWORD=${CF_PASSWORD}" \
+	--set "env.CF_DOMAIN=${CF_DOMAIN}"
+```
+
+Eventually you should see two pods start in the `SIDECAR_NAMESPACE`:
+
```

Comment:
There should be a third `setup` pod, running temporarily (errand).
Not sure if it makes sense to note that.
That said, if the `setup` errand fails not knowing that that pod was supposed to be there might be confusing.

Reply:
Good point. Added a detail about that in 2d8ca8e.